README.md

```diff
@@ -1,7 +1,128 @@
+- Target Metric 
+  - Top1 Accuracy > 80% (Note that some existing SotA models might not achieve this, like 
+    original ResNet, but this serve as a guideline for newly added models)
+
+### Object Detection
+- Dataset to evaluate
+  - [Coco 2017](https://www.tensorflow.org/datasets/catalog/coco#coco2017)
+  - [Open image v4](https://www.tensorflow.org/datasets/catalog/open_images_v4)
+  - [PASCAL voc](https://www.tensorflow.org/datasets/catalog/voc)
+- Target Metric
+  - COCO metric average precision (AP)
+- Detection model
+  - MaskRCNN
+  - YOLO (v3 or v4)
+  - RetinaNet
+- Other related utilities
+  - **ROI** (region of interest) layer (faster RCNN and Mask RCNN)
+  - Bounding box utils
+  - **RPN** (Region Proposal Network) used by faster RCNN and mask RCNN for bounding box 
+    generation.
+
+## Overlapping between keras.application and Keras-CV
+As you might notice, some area we would like to focus has overlapping with existing keras.application,
+and we would like to clarify the reason of the potential code duplication here.
+
+keras.application has a strong API contract that we can't easily make any backward incompatible
+change. On the other hand, there are a few of behavior we actually would like to change within the keras-cv
+  - Fetch imagenet weights by default, which might not want to do for keras-cv.
+  - Have the image preprocessing logic outside the model. We want to include all the preprocessing
+    within the model, so that user won't need to call the extra preprocessing methods.
+  - Only end-to-end model are exposed as public API, and Keras CV would like to include more building 
+    blocks as well.
+  - Some legacy model architectures are no longer commonly used and can't reach SotA performance, which
+    we might want to deprecate.
+
+Once the keras-cv applications are mature enough, we would add a deprecation warning in the 
+keras.applications code.
+
+
+## Community contribution guideline
+We would like to leverage/outsource the Keras community not only for bug reporting or fixes, 
+but also for active development for feature delivery. To achieve this, we will have the predefined
+process for how to contribute to this repository.
+
+The ideal workflow will be:
+
+- Keras defines the API interface for a certain task, eg a layer, model, dataset, metrics, etc. 
+- Post it to Github as a feature request. 
+- Have a hotlist/tag for those items and advocate them to the community.
+- Let community members claim the task they would like to work on.
+
+In the early stage, Keras team members will just claim those tasks by themselves. When there are 
+enough active community members, we should spend more time as reviewers rather than 
+authors. This should apply to bug fixes as well.
+
+User requested features will be carefully evaluated by keras team member before acceptance. 
+Based on the existing experience from tf-addons, even with a guideline of paper ref count > 50, 
```

Comment:
I think that Keras previously had also its own history other then tf-addons:

https://github.com/keras-team/keras-contrib

But again it is not clear where we want to route the limited contributors resources.

Reply:
I don't think we will put resource to keras-contrib anymore (I am not even sure why it exists in the first place, with tf.contrib deprecated/removed in tf2.0).

I will check with @fchollet  and see if we can just remove that repository.

Reply:
Yes I know we already discussed this on July:
https://github.com/tensorflow/community/pull/376
My point here was more about what the keras-contrib experience was and so that you can really rely also on the keras-contrib experience/history. 

Then we know some of the components of that repos was migrated in TFAddons like `tf.contribt`.

I still think that we could strongly evaluate if we could find a new role for TFAddons or we could just scheduling a migration of the community and the interesting components we currently have in Keras-* before we are going to create to much duplicates over the time in these two repo (I suppose that keras-nlp  would have a similar rationale as the one under review here).

Reply:
Sure. It will make a lot of sense to discuss with TFA for any of the existing components and see if they are good fit for keras-cv/nlp so that we can avoid any duplication.